src/test/java/com/facebook/openwifirrm/optimizers/tpc/LocationBasedOptimalTPCTest.java

```diff
@@ -103,41 +109,34 @@ void testLocationBasedOptimalTPCSuccessCase() throws Exception {
 		deviceDataManager.setDeviceApConfig(deviceC, apCfgC);
 
 		DataModel dataModel = new DataModel();
-		dataModel.latestDeviceStatus.put(
-			deviceA, TestUtils.createDeviceStatus(band, 36)
-		);
-		dataModel.latestState.put(
-			deviceA, TestUtils.createState(36, 20, dummyBssid)
-		);
-		dataModel.latestDeviceStatus.put(
-			deviceB, TestUtils.createDeviceStatus(band, 36)
-		);
-		dataModel.latestState.put(
-			deviceB, TestUtils.createState(36, 20, dummyBssid)
-		);
-		dataModel.latestDeviceStatus.put(
-			deviceC, TestUtils.createDeviceStatus(band, 36)
-		);
-		dataModel.latestState.put(
-			deviceC, TestUtils.createState(36, 20, dummyBssid)
-		);
+		for (String device : Arrays.asList(deviceA, deviceB, deviceC)) {
+			dataModel.latestDeviceStatus.put(device,
+				TestUtils.createDeviceStatus(UCentralConstants.BANDS));
+			dataModel.latestState.put(device,
+				TestUtils.createState(
+					UCentralUtils.LOWER_CHANNEL_LIMIT
+						.get(UCentralConstants.BAND_2G),
+					DEFAULT_CHANNEL_WIDTH, DEFAULT_TX_POWER,
+					UCentralUtils.LOWER_CHANNEL_LIMIT
+						.get(UCentralConstants.BAND_5G),
+					DEFAULT_CHANNEL_WIDTH, DEFAULT_TX_POWER, dummyBssid));
```

Comment:
formatting: see other comments - one argument per line (since it doesn't fit) and closing paren on a separate line. also, would define variables for `UCentralUtils.LOWER_CHANNEL_LIMIT.get(...)` to make this easier to read.